### PR TITLE
feat: make new javascript live

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [ ] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [ ] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Comment by naomi-lgbt** [comment]
GH wouldn't let me push to #52616 so here's a fresh pr

File: client/i18n/locales/english/translations.json [hunk 1]
```diff
@@ -125,6 +125,7 @@
     "professional-certs-heading": "Earn free professional certifications:",
     "interview-prep-heading": "Prepare for the developer interview job search:",
     "upcoming-heading": "Upcoming curriculum:",
+    "legacy-certs-heading": "Legacy certifications:",
```
Comment (moT01) [inline client/i18n/locales/english/translations.json:128]:
```suggestion
    "legacy-curriculum-heading": "Explore our Legacy Curriculum:",
```

File: client/src/components/Map/index.tsx [hunk 2]
```diff
@@ -127,6 +127,17 @@ function Map({ forLanding = false }: MapProps): React.ReactElement {
           <MapLi key={i} superBlock={superBlock} landing={forLanding} />
         ))}
       </ul>
+      <Spacer size='medium' />
+      <h2 className={forLanding ? 'big-heading' : ''}>
+        {t('landing.legacy-certs-heading')}
```
Comment (moT01) [inline client/src/components/Map/index.tsx:132]:
```suggestion
        {t('landing.legacy-curriculum-heading')}
```

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (naomi-lgbt) [reply client/i18n/locales/english/translations.json:128]: The legacy JS cert is remaining claimable, though.

Comment (moT01) [reply client/i18n/locales/english/translations.json:128]: hmm, but we're going to have the Legacy RWD, and Python for Everybody in there - those don't have certs.

Comment (naomi-lgbt) [reply client/i18n/locales/english/translations.json:128]: Do we not want to make a new Scientific Python cert? Or are the projects the same?

File: shared/config/superblocks.ts [hunk 3]
```diff
@@ -75,10 +76,10 @@ export const superBlockOrder: SuperBlockOrder = {
     SuperBlocks.CodingInterviewPrep,
     SuperBlocks.ProjectEuler
   ],
+  [SuperBlockStages.VisibleLegacy]: [SuperBlocks.JsAlgoDataStruct],
```
Comment (moT01) [inline shared/config/superblocks.ts:79]: I would prefer to just put this in the existing legacy and show the RWD with this - the Python for Everybody will be in there too.

File: client/i18n/locales/english/translations.json (hunk 1 above)
Comment (moT01) [reply client/i18n/locales/english/translations.json:128]: the projects are the same

**Review by moT01 — COMMENTED** [review]
This looks real good 🎉 I was able to claim both certs 🚀 

I probably didn't give it the time it needs, but I looked over it pretty good. I left a few comments below. The only other thing I didn't like is that the new cert says "Beta" and the old one still has the regular title. We can change that later I think.

**Review by jdwilkin4 — APPROVED** [review]
Was able to poke around a little bit locally and things are working on my end

Thank you for all of your hard work on this 🥳

**Comment by moT01** [comment]
Do we want to fix these failing tests here?

**Review by moT01 — APPROVED** [review]

**Pull request merged**